sdk/devcenter/azure-developer-devcenter/azure/developer/devcenter/_operations/_operations.py

```diff
@@ -4155,56 +3701,63 @@ def get_action(
 
         cls: ClsType[JSON] = kwargs.pop("cls", None)
 
-        request = build_dev_boxes_get_action_request(
+        _request = build_dev_center_get_dev_box_action_request(
             project_name=project_name,
+            user_id=user_id,
             dev_box_name=dev_box_name,
             action_name=action_name,
-            user_id=user_id,
             api_version=self._config.api_version,
             headers=_headers,
             params=_params,
         )
         path_format_arguments = {
             "endpoint": self._serialize.url("self._config.endpoint", self._config.endpoint, "str", skip_quote=True),
         }
-        request.url = self._client.format_url(request.url, **path_format_arguments)
+        _request.url = self._client.format_url(_request.url, **path_format_arguments)
 
-        _stream = False
+        _stream = kwargs.pop("stream", False)
         pipeline_response: PipelineResponse = self._client._pipeline.run(  # pylint: disable=protected-access
-            request, stream=_stream, **kwargs
+            _request, stream=_stream, **kwargs
         )
 
         response = pipeline_response.http_response
 
         if response.status_code not in [200]:
+            if _stream:
+                response.read()  # Load the body in memory and close the socket
             map_error(status_code=response.status_code, response=response, error_map=error_map)
             raise HttpResponseError(response=response)
 
-        if response.content:
-            deserialized = response.json()
+        if _stream:
+            deserialized = response.iter_bytes()
         else:
-            deserialized = None
+            if response.content:
+                deserialized = response.json()
+            else:
+                deserialized = None
 
         if cls:
-            return cls(pipeline_response, cast(JSON, deserialized), {})
+            return cls(pipeline_response, cast(JSON, deserialized), {})  # type: ignore
 
-        return cast(JSON, deserialized)
+        return cast(JSON, deserialized)  # type: ignore
 
     @distributed_trace
-    def skip_action(  # pylint: disable=inconsistent-return-statements
-        self, project_name: str, dev_box_name: str, action_name: str, user_id: str = "me", **kwargs: Any
+    def skip_dev_box_action(  # pylint: disable=inconsistent-return-statements
```

Comment:
I'm curious what you think about the updated naming @annatisch?

Reply:
It needs to be like that because we updated from `client.dev_boxes.skip_action` to `client.skip_dev_box_action`